### PR TITLE
[Windows upgrade e2e] config the tests

### PR DESCRIPTION
**Pull request by dntczdx**

### CODE REVIEW

File: config/testgrids/config.yaml [hunk 1]
```diff
@@ -204,6 +204,15 @@ dashboards:
         value: 'ci-ovf-import-e2e-tests-daily issue: <test-name>'
       - key: body
         value: <test-url>
+  - name: ci-windows-upgrade-tests
```
Comment (chases2) [inline config/testgrids/config.yaml:207]: Please add this in the `/testgrids/conformance/conformance-all.yaml` file, with the dashboard that uses it.

Comment (dntczdx) [reply config/testgrids/config.yaml:207]: I think it has been added there. Did you review the latest commit?

**Comment by spiffxp** [comment]
I'm confused why these are showing up in conformance, I don't think they have anything to do with conformance, but I'll save that for elsewhere

**Pull request merged**